### PR TITLE
DT-638: Make out of date grow darker the more out of date the app is

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: jobs/health.rb [hunk 1]
```diff
@@ -222,7 +222,18 @@ def gather_health_data(server)
 end
 
 def add_outofdate(version, check_version)
-  {outofdate: version != check_version}
+  if version == check_version
+    {outofdate: 0}
```
Comment (petergphillips) [inline jobs/health.rb]: copes with nomis api returning `UP` too

File: jobs/health.rb [hunk 2]
```diff
@@ -222,7 +222,18 @@ def gather_health_data(server)
...
+    else
+      begin
+        version_as_date = Date.parse(version)
+        check_version_as_date = Date.parse(check_version)
+        days_out_of_date = (check_version_as_date - version_as_date).to_i
+        {outofdate: (days_out_of_date + 3) * 8 }
```
Comment (petergphillips) [inline jobs/health.rb]: add initial step of 3 days so that we can differentiate

File: jobs/health.rb [hunk 3]
```diff
@@ -222,7 +222,18 @@ def gather_health_data(server)
...
+      rescue
+        {outofdate: 70 }
```
Comment (petergphillips) [inline jobs/health.rb]: default random colour if something goes wrong parsing the date

**Comment by Mjwillis** [comment]
Lgtm

**Pull request merged**